proxies.ts

```diff
@@ -9,12 +9,10 @@ export const proxies: ProxyAddresses = {
   polygon: "0x5c4e6A9e5C1e1BF445A062006faF19EA6c49aFeA",
   // tableland testnet testnets
   "ethereum-goerli": "0xDA8EA22d092307874f30A1F277D1388dca0BA97a",
-  "optimism-kovan": "0xf2C9Fc73884A9c6e6Db58778176Ab67989139D06",
```

Comment:
Removing this here also removes it from the compiled `proxies.js`. This means downstream consumers of proxies.js (namely the SDK) will need a new version to pull the updated proxies from.